Modules/Forum/classes/class.ilForumPostDraft.php

```diff
@@ -384,6 +384,31 @@ protected static function readDrafts($user_id)
             self::$instances[$user_id]['draft_ids'][$tmp_obj->getDraftId()] = $tmp_obj;
         }
     }
+
+    public static function readSortedDrafts($user_id, $thread_id, $sorting = ilForumProperties::VIEW_DATE_ASC)
```

Comment:
Please add PHP 7 typehints and return type declarations. Try to use https://psalm.dev/docs/annotating_code/type_syntax/array_types/ if a method returns an arrray.

Is there any way to avoid this `static` method here? Is it possible to use an object instead and call an instance-method?